### PR TITLE
[Synthetics] Clarify use of config_variable

### PR DESCRIPTION
**Pull request by pcothenet**
The example for using a global variable was confusing. This hopefully clarifies it.

### CODE REVIEW

**Review by zippolyte — CHANGES_REQUESTED** [review]
Hey, thanks for the contribution.
However, docs are generated using `tfplugindocs`, so if you could make that change in http://github.com/datadog/terraform-provider-datadog/blob/57bb1d93f45eabbd8fc824f46775e85c9c8563bc/templates/resources/synthetics_test.md.tmpl instead, and then run `make docs`, that'd be great.

File: templates/resources/synthetics_test.md.tmpl [hunk 1]
```diff
@@ -14,16 +14,25 @@ description: |-
 #### *Warning*
 Starting from version 3.1.0+, the direct usage of global variables in the configuration is deprecated, in favor of
 local variables of type `global`. As an example, if you were previously using `{{ "{{ GLOBAL_VAR }}" }}` directly in your
-configuration, you now need to add a `configVariable` of type `global` and whose `id` must be the `id` 
-of the global variable `GLOBAL_VAR` (you can find it in Synthetics UI) and whose name can be chosen freely. 
+configuration, you now need to add a `config_variable` of type `global` and whose `id` must be the `id`
```
Comment (ruthnaebeck) [inline templates/resources/synthetics_test.md.tmpl:17]:
```suggestion
configuration, add a `config_variable` of type `global` with the `id` matching the `id` 
```

File: templates/resources/synthetics_test.md.tmpl [hunk 2]
```diff
@@ -14,16 +14,25 @@ description: |-
 #### *Warning*
 Starting from version 3.1.0+, the direct usage of global variables in the configuration is deprecated, in favor of
 local variables of type `global`. As an example, if you were previously using `{{ "{{ GLOBAL_VAR }}" }}` directly in your
-configuration, you now need to add a `configVariable` of type `global` and whose `id` must be the `id` 
-of the global variable `GLOBAL_VAR` (you can find it in Synthetics UI) and whose name can be chosen freely. 
+configuration, you now need to add a `config_variable` of type `global` and whose `id` must be the `id` 
+of the global variable `GLOBAL_VAR` (you can find it in the Synthetics UI or from the output of the `datadog_synthetics_global_variable` resource) and whose name can be chosen freely.
```
Comment (ruthnaebeck) [inline templates/resources/synthetics_test.md.tmpl:18]:
```suggestion
of the global variable `GLOBAL_VAR`, which can be found in the Synthetics UI or from the output of the `datadog_synthetics_global_variable` resource. The name can be chosen freely. 
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Docs review

**Pull request closed without merging**